### ISSUE TITLE
I created an internal version of this repo

### ISSUE THREAD

**Issue by claytonpbarrows** [NREL/MAGMA#12]
@NREL/nrel-plexos-users I created an NREL internal version of this repository in addition to this one.
NREL employees, please run the following commands from your local repository or submodule directory for this repository:

add remote
```bash
git remote add internal git@github.nrel.gov:PCM/PLEXOS-Vis.git
```

to push externally to this github.com repository (note that ```<branch name>``` is usually ```origin```
```bash
git push origin <branch name>
```

to push internally to the github.nrel.gov repo
```bash
git push internal <branch name>

**Comment by claytonpbarrows** [comment]
made repo public

**Comment by githubgbs** [comment]
I was just about ready to write you and say thank you…

-Greg

From: claytonpbarrows
Reply-To: NREL/MAGMA
Date: Wednesday, August 24, 2016 at 11:41 AM
To: NREL/MAGMA
Subject: Re: [NREL/MAGMA] I created an internal version of this repo (#12)


Closed #12<https://github.com/NREL/MAGMA/issues/12>.
